include/gsl/multi_span

```diff
@@ -79,7 +79,7 @@ namespace details
     template <typename SizeType>
     struct SizeTypeTraits
     {
-        static const SizeType max_value = std::numeric_limits<SizeType>::max();
+        static const SizeType max_value = (std::numeric_limits<SizeType>::max)();
```

Comment:
Hmm, you don't really want to transmoglify the source code like that.
There are other ways to work around this.  For example, add `-DNOMINMAX` to the command line when compiling and you know you're dragging `windows.h` into the source code.